code/lib/cli-storybook/src/doctor/getIncompatibleStorybookPackages.ts

```diff
@@ -112,12 +126,24 @@ export const getIncompatiblePackagesSummary = (
       )} as they depend on different major versions of Storybook packages:`
     );
     incompatiblePackages.forEach(
-      ({ packageName: addonName, packageVersion: addonVersion, homepage, availableUpdate }) => {
+      ({
+        packageName: addonName,
+        packageVersion: addonVersion,
+        homepage,
+        availableUpdate,
+        packageStorybookVersion,
+      }) => {
         const packageDescription = `${picocolors.cyan(addonName)}@${picocolors.cyan(addonVersion)}`;
         const updateMessage = availableUpdate ? ` (${availableUpdate} available!)` : '';
+        const dependsOnStorybook =
+          packageStorybookVersion != null
+            ? ` which depends on ${picocolors.red(packageStorybookVersion)}`
```

Comment:
like mentioned before the `packageStorybookVersion` could be related to any of the core packages, not just the `storybook` package